src/main/java/br/com/ks/viagem/emvoos/controllers/PaisController.java

```diff
@@ -0,0 +1,52 @@
+package br.com.ks.viagem.emvoos.controllers;
+
+import br.com.ks.viagem.emvoos.DTOs.request.PaisRequest;
+import br.com.ks.viagem.emvoos.DTOs.response.PaisResponse;
+import br.com.ks.viagem.emvoos.repositories.PaisRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.springframework.http.HttpStatus.CREATED;
+
+@RestController
+@RequestMapping("api/v1/pais")
+public class PaisController {
+
+    private PaisRepository paisRepository;
+
+    @Autowired
+    public PaisController(PaisRepository paisRepository) {
+        this.paisRepository = paisRepository;
+    }
+
+    @ResponseStatus(CREATED)
+    @PostMapping(value = "/salvar",
+            produces = MediaType.APPLICATION_JSON_VALUE,
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Aqui é só uma observação... O default já é responder e receber como JSON(através do jackson), quando configuramos o starter para web. Aí nem precisa definir aqui... mas é só um detalhe.